styleguide/_themes/global/scss/swatch.scss

```diff
@@ -36,3 +36,9 @@ $brand-danger:      $rackspace-red !default;
 
 //alpha colors
 $black-shadow:		rgba($black, .175);
+
+$clickablebox-shadow: rgba($black, .06);
+$clickablebox-hover: rgba($black, .2);
+$top-overlay-shadow: rgba($black, .15);
```

Comment:
It would be preferable to use a scaled naming convention here like we do for grays above.  
Something like:
`$black-shadow-dark`
`$black-shadow-darker`
`$black-shadow-darkest`
etc...